lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -1302,6 +1298,36 @@ CommandObject *CommandInterpreter::GetUserCommandObject(
   return {};
 }
 
+CommandObject *CommandInterpreter::GetAliasCommandObject(
+    llvm::StringRef cmd, StringList *matches, StringList *descriptions) const {
+  std::string cmd_str(cmd);
+  auto find_exact = [&](const CommandObject::CommandMap &map) {
```

Comment:
Nit:
```suggestion
  auto find_exact = [&](const CommandObject::CommandMap &map) -> CommandObject * {
```
Then you don't need the casts below.